### DISCUSSION TITLE
Stability Matrix via Local Network

### DISCUSSION THREAD

**Discussion by sharabhi55** [LykosAI/StabilityMatrix#426] [General]
Just wondering if there is a way to access stability matrix interface on my local network?

If not, what is the best way to setup each webui app to work over the my network?

**Comment by bnwa** [comment]
This is the number one feature I want from StabilityMatrix, I hate having to use Windows to admin Stability/models and would like to minimize it's use as far as possible.

**Comment by mtschill** [comment]
My workaround is Tailscale, and then I use the serve command to broadcast the local address. I can access it from any other device in my tailnet.

**Reply by bnwa** [reply]
I'm confused, are you referring to the local address of a particular package, e.g. ComfyUI; or are you referring to StabilityMatrix itself and there's a detail being missed by others that the UI is served over a localhost address with a specific port? If the latter, I wish they'd expose that and the port so we can directly assign `0.0.0.0:8000` or whatever. That would be the intent of this issue.
